demisto_sdk/commands/format/format_module.py

```diff
@@ -284,9 +283,9 @@ def run_format_on_file(input: str, file_type: str, from_version: str, interactiv
     if file_type not in ('integration', 'playbook', 'script') and 'add_tests' in kwargs:
         # adding tests is relevant only for integrations, playbooks and scripts.
         del kwargs['add_tests']
-    if file_type != FileType.INCIDENT_FIELD.value and 'id_set_path' in kwargs:
-        # relevant only for incidentfield
-        del kwargs['id_set_path']
+    # if file_type != FileType.INCIDENT_FIELD.value and 'id_set_path' in kwargs:
+    #     # relevant only for incidentfield
+    #     del kwargs['id_set_path']
```

Comment:
This should be removed but left as comment for now in case there will be any related errors before merging